packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPendingRow/components/TransactionsPendingRowField/TransactionsPendingRowField.tsx

```diff
@@ -26,24 +26,21 @@ const TransactionsPendingRowField: React.FC<TProps> = ({ className, hint, name,
     const isFieldHovered = useHover(fieldRef);
 
     const onValueClick = useCallback(() => {
-        if (hint)
-            isMobile
```

Comment:
Refactored TransactionsPendingRowField to remove unneeded checks as `isMobile` check is already done on the `onValueClick` button